packages/panels/src/components/sidebar-menu/SidebarMenu.tsx

```diff
@@ -24,15 +25,21 @@ export const SidebarMenu: React.FC<SidebarMenuProps> = ({
   className,
   children,
   onCloseClick,
+  hideCloseButton,
```

Comment:
Set the default value to `false`. Not necessary, but neat.

```suggestion
  hideCloseButton = false,
```